pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -84,7 +84,7 @@ public static String jsonFormat(Object object)
   /**
    * Extract object based on Json path
    */
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true)
```

Comment:
No need to annotate this function because when input is `null`, output is also `null`. The null check on line 92 can actually be removed with the change in this PR because that will never be hit